src/components/TrackTile/TrackerDetails/UnitPicker.tsx

```diff
@@ -87,10 +87,7 @@ export const UnitPicker: FC<UnitPickerProps> = (props) => {
                   variant="semibold"
                   style={styles.unitPickerPopupAccessoryDoneTextIOS}
                 >
-                  {i18n.t('f92965e2c8a7afb3c1b9a5c09a263636', {
```

Comment:
I would expect to see this changes across all the language JSON files too. Usually I do a Replace All when updating the key.

Reply:
I really appreciate/prefer PRs focused on one thing.  If we take the "update all i18next keys" (whether soon or as a final wrap-up PR), it should be especially focused on that one thing because it'll be so large.

Reply:
Correct.  I have a singular focused PR upcoming with all fixed.  This one was required for the test to pass.

Reply:
Updating the key name in these files would only change a single string across 7 files and usually it is easier to update these in the JSON files while you still have the original key. Also the lookups to the translations for this key are broken until the updates are made but that too probably isn't a big deal given we will just fallback to the default text.

Reply:
Thanks for clarifying Bruce - I misunderstood.  I thought you were suggesting we update **all** the i18n keys at once, and I got on a soapbox about PR focus 😆.  You're spot on- @namick since we imported track-tile-ui's translation (JSON) files, the string `f92965e2c8a7afb3c1b9a5c09a263636` is throughout those files and orphaned once this PR merges.  Assuming you already know that, and this is all just me catching up after butting in, all good 👍.  i.e. It's fine whether we update `f92965e2c8a7afb3c1b9a5c09a263636` -> `track-tile-done` in this PR or a subsequent one, as long as we know we're getting to it soon.